### DISCUSSION TITLE
Nuxt 4 release

### DISCUSSION THREAD

**Discussion by gurusamy** [nuxt/nuxt#25182] [Questions]
It appears that Nuxt 4 is slated for release in Q1 2024. Are there significant fundamental changes expected in Nuxt 4 compared to Nuxt 3?

**Comment by TheAlexLichter** [comment]
Nothing as big as Nuxt 2 -> Nuxt 3, no worries 😋

**Reply by MuhammadM1998** [reply]
Is there's a roadmap available for Nuxt 4? I'm curious if Nuxt Auth is included or not 👀

**Reply by huang-julien** [reply]
Nuxt Auth is separated from the core. So both probably won't be released together

**Reply by MuhammadM1998** [reply]
Okay thanks!

**Reply by EddyDezuraud** [reply]
I can't wait to see the roadmap! 👀
